### ISSUE TITLE
Option to have the editor track the current flow during simulation

### ISSUE THREAD

**Issue by rowanseymour** [nyaruka/floweditor#742]
Maybe this is a little nuts..  but you can imagine having an option on the simulator that tells the editor to switch to the current flow when it's processing a simulator response... that way you can do a full integration test of your flows, and watch the contact's progress thru each different subflow.

**Comment by ericnewcomer** [comment]
I don't think it would necessarily be a lot of work to make happen. We'd really need to be mindful about how that is presented as I could see it getting pretty confusing. But generally, I think this is a good idea and it is also something I've contemplated.
